AUTHORS

```diff
@@ -0,0 +1,12 @@
+<!--
+Copyright (C) Sergio Durigan Junior <sergiodj@debian.org>, et al.
```

Comment:
We have to update every file in the project (including changing this line) to match what we are using in LICENSE, so it should be:
`Copyright (C) Samuel Henrique <samueloph@debian.org>, Sergio Durigan
Junior <sergiodj@debian.org> and many contributors, see the AUTHORS
file.`

Right?